Formula/fastlane.rb

```diff
@@ -19,7 +20,7 @@ class Fastlane < Formula
     sha256 "b9e0be03c7c28d5d11d274ae8c0b20a3d96e5619d9bdb6fca788d312a2365070" => :mojave
   end
 
-  depends_on "ruby"
+  depends_on "ruby@2.6"
```

Comment:
retrospectively thinking, I wonder if we should add ruby@2.7 and have the formula depends on it.